CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `2.15.0`
   - withdraw all QR codes when one is withdrawn
   - do not create a group if the sender includes self in the To field
+- minor visual improvements to keyboard shortcut preview #5495
```

Comment:
I don't think these changes should add a changelog entry...

Reply:
my rule of thumb was here:
- user visible && different to last released version